.gitmodules

```
@@ -139,10 +136,7 @@
 [submodule "dependencies/git/ngx_http_redis"]
 	path = dependencies/git/ngx_http_redis
 	url = https://github.com/osokin/ngx_http_redis
-[submodule "dependencies/git/nginx-opentracing"]
-	path = dependencies/git/nginx-opentracing
-	url = https://github.com/opentracing-contrib/nginx-opentracing
-[submodule "dependencies/git/apicast-nginx-module"]
+[submodule "dependencies/git/apicas-nginx-module"]
```

(I can't believe I noticed this, but) we have a typo here, right?

```suggestion
[submodule "dependencies/git/apicast-nginx-module"]
```

---

yes typo, I haven't checked any Konflux build but the build should fail right :thinking: ?

---

Oh I see, it came from my commit, I was confused by the two block quotes. I need more coffee :coffee:

---

fixed